src/types.rs

```diff
@@ -85,8 +87,9 @@ pub struct Affected {
 }
 
 impl Affected {
-    pub fn new(package: &str) -> Self {
+    pub fn new(package: &str, avg: &Avg) -> Self {
```

Comment:
This isn't really correct. The way the logic works is that `Affected` is a bundle of all CVEs from all AVGs related to a certain package. Hence the CVEs list etc gets extended. Declaring one Avg for this bundle is not right, it should be a Vec of Avg's/names

Reply:
I've updated the PR so `--json` prints the filtered AVGs instead of the `Affected` bundle.